omod/src/main/webapp/resources/htmlforms/Vaccination.xml

```diff
@@ -204,685 +79,58 @@ hide.hidden {
-            jQuery('#pentafield3').fadeOut(0);
-         }
-       }
-	 
-	   // For DiTePer
-	   if(!getValue('diteper1.date'))
-	   {
-			if ((window.patientAgeDays*86400000) > window.diteper1MinAge) {
-				jQuery('#diteperfield1').fadeOut(0);
-			 }
-		}
-		if(getValue('diteper1.date'))
-		{
-			jQuery('#diteperfield1').fadeIn(0);
-		}
-		if (!getValue('diteper2.date') ) {
-         if ( !getValue('diteper1.date') || (Math.abs(new Date() - new Date(getValue('diteper1.date'))) &lt; window.diteper2Spacing) || !(((window.patientAgeDays*86400000) > window.diteper2MinAge &amp;&amp; (window.patientAgeDays*86400000) &lt; (window.diteper2MaxAge-window.retrospectiveTime)) ) ) {
-            jQuery('#diteperfield2').fadeOut(0);
-         }
-      }
-      if (!getValue('diteper3.date') ) {
-         if ( !getValue('diteper1.date') || !getValue('diteper2.date') || (Math.abs(new Date() - new Date(getValue('diteper2.date'))) &lt; window.diteper3Spacing) || !(((window.patientAgeDays*86400000) > window.diteper3MinAge &amp;&amp; (window.patientAgeDays*86400000) &lt; (window.diteper3MaxAge-window.retrospectiveTime)) ) ) {
-            jQuery('#diteperfield3').fadeOut(0);
-         }
-       }
-	   if (!getValue('diteper4.date') ) {
-         if ( !getValue('diteper1.date') || !getValue('diteper2.date') || !getValue('diteper3.date') || (Math.abs(new Date() - new Date(getValue('diteper3.date'))) &lt; window.diteper4Spacing)) {
-            jQuery('#diteperfield4').fadeOut(0);
-         }
-      }
-	   //For HIB
-	   if ( !(getValue('hib1.date') || ((window.patientAgeDays*86400000) > window.hib1MinAge &amp;&amp; (window.patientAgeDays*86400000) &lt; (window.hib1MaxAge+window.retrospectiveTime)) ) ) {
-            jQuery('#hibfield1').fadeOut(0);
-         }
-      if (!getValue('hib2.date') ) {
-         if ( !getValue('hib1.date') || (Math.abs(new Date() - new Date(getValue('hib1.date'))) &lt; window.hib2Spacing) || !(((window.patientAgeDays*86400000) > window.hib2MinAge &amp;&amp; (window.patientAgeDays*86400000) &lt; (window.hib2MaxAge-window.retrospectiveTime)) ) ) {
-            jQuery('#hibfield2').fadeOut(0);
-         }
-      }
-      if (!getValue('hib3.date') ) {
-         if ( !getValue('hib1.date') || !getValue('hib2.date') || (Math.abs(new Date() - new Date(getValue('hib2.date'))) &lt; window.hib3Spacing) || !(((window.patientAgeDays*86400000) > window.hib3MinAge &amp;&amp; (window.patientAgeDays*86400000) &lt; (window.hib3MaxAge-window.retrospectiveTime)) ) ) {
-            jQuery('#hibfield3').fadeOut(0);
-         }
-       }
-	   if (!getValue('hib4.date') ) {
-         if ( !getValue('hib1.date') || !getValue('hib2.date') || !getValue('hib3.date') || (Math.abs(new Date() - new Date(getValue('hib3.date'))) &lt; window.hib4Spacing)) {
-            jQuery('#hibfield4').fadeOut(0);
-         }
-      }
-	  //Fin HIB
-	  //For ROR
```

Comment:
i fear for these big deletions, did you intend to remove all these @jamesfeshner ?